spec/features/reservations/traveler_can_make_a_reservation_spec.rb

```diff
@@ -10,6 +10,7 @@
 =end
 
 require "rails_helper"
+include ModelHelpers
```

Comment:
I thought so too... but then I was getting errors for not having the `.to_date_picker_format` method defined.  Perhaps I have it setup wrong.  Another set of eyes would be much appreciated!